integration-tests/helpers/index.js

```diff
@@ -25,10 +25,13 @@ async function runAndCheckOutput (filename, cwd, expectedOut) {
       out = Buffer.concat([out, data])
     })
     proc.stderr.pipe(process.stdout)
-    proc.on('exit', () => resolve(out.toString('utf8')))
-    setTimeout(() => {
+    const exitTimeout = setTimeout(() => {
       if (proc.exitCode === null) proc.kill()
-    }, 1000) // TODO this introduces flakiness. find a better way to end the process.
+    }, 10000) // TODO this introduces flakiness. find a better way to end the process.
```

Comment:
Why did this have to be increased?

Reply:
It doesn't. Latest rev doesn't touch this number.